web/src/components/Upload_Confirmation/index.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from 'react'
+import {
+  Message,
+  Icon,
+  Button,
+  Container,
+  Input,
+  Dropdown
+} from 'semantic-ui-react'
+import styled from 'theme'
+
+const StyledContainer = styled(Container)`
+  margin-top: 30px;
+`
+
+const ContentContainer = styled.div`
+  margin: 0px 130px 0px 130px;
+`
+
+const SuccessBanner = styled(Message)`
+  background: #619e54 !important;
+  height: 48px;
+  width: 100%;
+  display: flex;
+  border-radius: 4px;
+  align-items: center;
+  margin-bottom: 32px;
+  * {
+    color: ${({ theme }) => theme.colors.white};
+  }
+  i.icon {
+    font-size: 20px;
+  }
+  h3 {
+    font-size: 24px;
+  }
+`
+
+const SuccessIcon = () => <Icon name='check circle outline' />
+
+const SuccessMessage = styled.span`
+  font-family: ${({ theme }) => theme.fonts.secondary};
+  font-size: 1rem;
+`
+
+const FormMessage = styled.h3`
+  font-size: 1.5rem;
+`
+
+const FormContainer = styled.div`
+  width: 500px;
+  margin: 0 auto;
+`
+
+const FormContent = styled.div`
+  padding: 1.85rem 4.5rem 5rem 2rem;
+`
+const DisabledInput = styled(Input)`
+  min-width: auto;
+  width: 160px;
+`
+
+const CalendarIcon = () => <Icon name='calendar outline' />
+
+const DropdownInput = styled(Dropdown)`
+  min-width: auto !important;
+  width: 160px;
+  //eventually need to make borders of this dropdown #232735 per wireframe
+  // :focus-within {
+  //   border: 1px solid #232735 !important;
+  // }
+`
+
+const InputLabel = styled.p`
+  font-family: ${({ theme }) => theme.fonts.primary};
+  font-size .875rem;
+  line-height: 24px;
+  margin: 0px 0px 32px 0px !important;
+`
+const SubmitForm = styled.form`
+  display: flex;
+  justify-content: space-between;
+  margin-top: 48px;
+  button {
+    border-radius: 2px !important;
+    font-family: ${({ theme }) => theme.fonts.secondary} !important;
+    font-style: normal !important;
+    font-weight: 500 !important;
+    width: 160px !important;
+    font-size: 1rem !important;
+  }
+`
+
+const SaveButton = styled(Button)`
+  color: ${({ theme }) => theme.colors.white} !important;
+  background-color: rgba(53, 53, 53, 0.9) !important;
+`
+
+const CancelButton = styled(Button)`
+  color: rgba(53, 53, 53, 0.9) !important;
+  background-color: #f4f5f4 !important;
+`
+
+const options = [
```

Comment:
I would like to clarify whether the nomenclature is `Device A` or `Dustriak II`. However, I foresee this becoming data filled in by a dedicated `GET` request. These are issues for a future iteration.